odk1-src/aggregate-upgrade.rst

```diff
@@ -109,724 +109,19 @@ If you are unable to log onto your server, you will need to search for the versi
 General steps for upgrading
 ------------------------------------
 
-1. Disable all submissions to ODK Aggregate, in the :guilabel:`Form Management` tab.
-2. Use :doc:`ODK Briefcase  <briefcase-using>` to pull a copy of all data to your computer.
-3. Search upward in the :ref:`release notes <release-notes>` below, starting from your current ODK Aggregate version to locate the most recent ODK Aggregate version that does not require any manual upgrade steps. 
-4. Log onto your server to confirm that it is still functioning.
-5. Download the next ODK Aggregate version, perform the manual upgrade steps if any and upgrade to that version (and perform any post-install upgrade steps, if any).
-6. Log onto your server to confirm that it is still functioning.
-7. Repeat the steps 3-6 until you have upgraded to the current version.
-8. Enable submissions to ODK Aggregate via the :guilabel:`Form Management` tab.
+#. Disable all submissions to ODK Aggregate, in the :guilabel:`Form Management` tab.
+#. Use :doc:`ODK Briefcase  <briefcase-using>` to pull a copy of all data to your computer. 
+#. Log onto your server to confirm that it is still functioning.
+#. Download the next ODK Aggregate version and upgrade to that version.
```

Comment:
Maybe we should add a note why we say this. Something like: "Do not simply upgrade to the latest version of Aggregate. We recommend you incrementally upgrade because Aggregate releases may require manual changes to your database"

Reply:
I would also add a tip (or maybe a new section) on how to determine your Aggregate version. https://github.com/opendatakit/opendatakit/wiki/Aggregate-Release-Notes#determining-your-odk-aggregate-version has copy you can start from.

Reply:
I guess we can also add a tip that says if you want to setup a new instance, you can move using  Briefcase. The caveat is that you will lose your URL.

Reply:
> lose your URL

Only if you use the assigned url from your cloud provider, right?
If you've mapped a human-friendly URL to it, you can just move that to the new install right?

Reply:
Most people on App Engine don't have a URL they can remap, but yeah, others can.